db/migrate/20171110152106_rename_gng_categories.rb

```diff
@@ -0,0 +1,6 @@
+class RenameGngCategories < ActiveRecord::Migration
+  def change
+    Category.where(name: "gng1103").update_all(name: "gng1103/gng1503")
+    CategoryOption.where(name: "gng2101").update_all(name: "gng2101/gng1501")    
```

Comment:
I'm confused as to what's happening here. Shouldn't this be `Category.where(name: "gng2101").update_all(name: "gng2101/gng2501")  `